timescaledb/tutorials/nfl-fantasy-league.md

```diff
@@ -0,0 +1,188 @@
+# Win your NFL fantasy league with TimescaleDB
+
+This tutorial is a step-by-step guide on how to ingest and analyze american football data with TimescaleDB.
+
+The dataset that we're using is provided be the National Football League (NFL) and contains data about 
+all the passing plays of the 2018 NFL season. We're ingesting this dataset and exploring it in fun ways. You will also see how this dataset and using TimescaleDB can improve your NFL fantasy league strategy. If you aren't an NFL fan, don't worry, you will still find this tutorial helpful as it's showcasing how to get started with TimescaleDB and explore a real world dataset with SQL and Python.
```

Comment:
The second sentence feels a little out of place, maybe expand on things more? Or cut it??

Reply:
"You will also see how this dataset, and using TimescaleDB can improve your..." maybe a comma here?

Reply:
https://github.com/timescale/docs/pull/187/commits/6a12d2bc2512699e3004e5cd0a3da6bf3ce7a39f